test/crying.jl

```diff
@@ -19,8 +19,8 @@ let
     r = simulate(sim, problem, policy, updater(policy), ib, true)
     @test r ≈ -100.0 atol=0.01
 
-    # test generate_o
-    o = generate_o(problem, true, MersenneTwister(1))
+    # test gen(::o,...)
+    o = gen(DDNOut(:o), problem, true, MersenneTwister(1))
```

Comment:
DDNNode

Reply:
Is this because it is an initial observation? I thought solvers etc. should call `DDNOut`